P2946-throws_nothing/P2946.md

```diff
@@ -292,7 +299,7 @@ static_assert(std::is_same_v<decltype(g1), decltype(g4)>);
 ```
 
 Intentionally making `[[throws_nothing]]` invisible to the `noexcept` operator
-prevents using it to select an algorithm at compile time; the attribute does
+prevents it from being used to select an algorithm at compile time; the attribute does
```

Comment:
I suggest "prevents using `[[throws_nothing]]` to select..." Then you're being clear and still reducing words.